### ISSUE TITLE
'TestStreaming'(api_test.go) failed

### ISSUE THREAD

**Issue by spikeekips** [bosnet/sebak#137]
```
=== RUN   TestStreaming
--- FAIL: TestStreaming (2.41s)
	api_test.go:102: Expected:2001 Actual:2000
	api_test.go:102: Expected:2002 Actual:2001
	api_test.go:102: Expected:2004 Actual:2003
	api_test.go:102: Expected:2007 Actual:2006
	api_test.go:102: Expected:2011 Actual:2010
	api_test.go:102: Expected:2016 Actual:2015
	api_test.go:102: Expected:2022 Actual:2021
	api_test.go:102: Expected:2029 Actual:2028
	api_test.go:102: Expected:2037 Actual:2036
```

**Comment by kfangw** [comment]
@spikeekips which commit did you tested?

**Comment by kfangw** [comment]
TestStreaming is outdated name of the test function
Could you pull latest commit?
or please reset local branch to my branch(PR)?

**Comment by spikeekips** [comment]
@kfangw I tested under the current latest master, 56f7adc584b351bcc5c95407747222cc8a8dfa33 .

**Comment by spikeekips** [comment]
@kfangw Please let me know your PR :)

**Comment by kfangw** [comment]
@spikeekips https://github.com/bosnet/sebak/pull/121

**Comment by kfangw** [comment]
Solved by #121
